makelib/golang.mk

```diff
@@ -297,11 +305,18 @@ $(DEP):
 	@$(OK) installing dep $(HOSTOS)-$(HOSTARCH)
 
 $(GOLANGCILINT):
-	@$(INFO) installing golangci-lint
+	@$(INFO) installing golangci-lint-v$(GOLANGCILINT_VERSION) $(HOSTOS)-$(HOSTARCH)
 	@mkdir -p $(TOOLS_HOST_DIR)/tmp-golangci-lint || $(FAIL)
-	@GOPATH=$(TOOLS_HOST_DIR)/tmp-golangci-lint GOBIN=$(TOOLS_HOST_DIR) $(GOHOST) get -u github.com/golangci/golangci-lint/cmd/golangci-lint || rm -fr $(TOOLS_HOST_DIR)/tmp-golangci-lint || $(FAIL)
+	@if [ "$(GOHOSTARCH)" = "arm64" ]; then\
```

Comment:
we should never get here because of https://github.com/upbound/build/blob/master/makelib/common.mk#L110

Reply:
Aha, that makes life easier. This was copy pasta from the dep install logic - should I clean that up too?

Reply:
sure.